src/pages/Admin/Charity/Withdraws/Withdrawer/WithdrawForm/Network/Network.tsx

```diff
@@ -0,0 +1,88 @@
+import { Listbox } from "@headlessui/react";
+import { useEffect } from "react";
+import { useFormContext } from "react-hook-form";
+import { WithdrawValues } from "../types";
+import { useChainsQuery } from "services/apes";
+import Icon, { DrawerIcon } from "components/Icon";
+import QueryLoader from "components/QueryLoader";
+
+export default function Network() {
+  const queryState = useChainsQuery({});
+  const { register, watch, trigger } = useFormContext<WithdrawValues>();
+  const network = watch("network");
+
+  useEffect(() => {
+    (async () => {
+      //validate address on network change
+      await trigger("beneficiary");
```

Comment:
this is not working anymore
expected behavior when destination and network is mistmatched
<img width="500" alt="image" src="https://user-images.githubusercontent.com/89639563/223012948-cab0a1c6-386f-4bc4-b656-d0ece080e4ac.png">

Reply:
Unable to repro. Can you retry this, maybe the issue was fixed with one of the recent commits?

Note: was able to repro for Terra, but this was due to missing `case` in `getWalletAddrPattern`. Now fixed.

Reply:
@ap-justin **does the issue still occur?**